views/project/billing.erb

```diff
@@ -142,6 +142,26 @@
                     }
                   ) %>
                 </div>
+                <div class="col-span-full">
```

Comment:
Can you make them to `sm:col-span-4`? This means half width. Full width seems a bit unappealing.

<img width="1189" alt="Screenshot 2024-02-28 at 21 08 46" src="https://github.com/ubicloud/ubicloud/assets/993199/8e2517cb-3cef-43c4-9544-d1f5cb3ea914">